microshift_networking/microshift-networking.adoc

```diff
@@ -26,9 +26,7 @@ include::modules/microshift-ovs-snapshot.adoc[leveloffset=+1]
 include::modules/microshift-mDNS.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
-[id="additional-resources_microshift-applying-networking-settings"]
+[id="additional-resources_microshift-understanding-networking-settings"]
 .Additional resources
 
-* xref:../microshift_troubleshooting/microshift-troubleshooting.adoc#microshift-version[Troubleshooting]
-* xref:../microshift_troubleshooting/microshift-troubleshooting.adoc#microshift-troubleshooting-nodeport[Troubleshooting the NodePort service]
-* xref:../microshift_troubleshooting/microshift-troubleshooting.adoc#microshift-nodeport-unreachable-workaround[NodePort unreachable workround]
+* To troubleshoot NodePort connection problems, read about the known issue in xref:../microshift_release_notes/microshift-4-13-release-notes.adoc#microshift-4-13-release-notes[Release Notes]
```

Comment:
Per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections), "Additional resources" bullets should more or less just be the name of the section/topic being linked to.

You could deep-link a little further the known issues section itself, but maybe fudge the title a little like:

```
* xref:../microshift_release_notes/microshift-4-13-release-notes.html#microshift-4-13-known-issues[{product-title} {product-version} release notes --> Known issues] 
```